### PR TITLE
Remote repo download files run

### PR DESCRIPTION
**Pull request by ahmadswaid**

### CODE REVIEW

File: de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/joiner/JoinerNodeModel.java
Comment (miguelalba) [inline de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/joiner/JoinerNodeModel.java:858]: There is a potential resource leak here as the stream may not be closed on IO errors. This warning should not be suppressed. Streams can be closed automatically in case of errors with try-with resources.

File: de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/joiner/JoinerNodeModel.java
Comment (miguelalba) [inline de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/joiner/JoinerNodeModel.java:865]: We should document the string array model with every value at each position. I cannot tell what data model[4], model[5], model[6] and others are holding.

**Review by miguelalba — CHANGES_REQUESTED** [review]

**Comment by schuelet** [comment]
@ahmadswaid Can you tell me which endpoint I need to put into the FSK DB View node?

**Comment by ahmadswaid** [comment]
> @ahmadswaid Can you tell me which endpoint I need to put into the FSK DB View node?

it should point to the hosting server:
if you metadata endpoint is running under: 
**http://127.0.0.1:8080/metadata** 
then the repository should be set to:
 **http://127.0.0.1:8080/**

File: de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/runner/RunnerNodeModel.java
Comment (schuelet) [inline de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/runner/RunnerNodeModel.java:404]: @AhmadrSwaid : Is there a reason you don't use the originalOutputParameters here anymore or was it not working?

Comment (ahmadswaid) [reply de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/runner/RunnerNodeModel.java:404]: Yes, the JoinRelation holds the global unique parameter Id and I used the originalOutputParameters as indicator to which relation are relevant to the current joined part.

**Review by schuelet — APPROVED** [review]

**Comment by ahmadswaid** [comment]
I can't merge without @miguelalba  approval also

**Review by miguelalba — APPROVED** [review]

**Pull request merged**
